# Configuration Settings for the Rover Rx

device: tcp://xxx.xxx.xxx.xxx:xxxx

serial:
  baudrate: 115200
  rx_serial_port: USB1
  hw_flow_control: off

frame_id: gnss
ins_frame_id: INS


receiver_type: INS  #INS #GNSS

datum: ETRS89

marker_to_arp:
  delta_e: 0.0
  delta_n: 0.0
  delta_u: 0.0
  
ant_type: Unknown
ant_serial_nr: Unknown

# INS configuration

attitude_offset:
  heading: 0.0
  pitch: 0.0

imu_orientation:
  sensor_default: false
  manual: true
  angles:
    theta_x: 180.0
    theta_y: 0.0
    theta_z: 0.0

ins_point_of_interest:
  poi_x: 0.0
  poi_y: 0.0
  poi_z: 0.0

ins_antenna_lever_arm:
  x: 0.70
  y: 0.0
  z: 0.0
  
ins_vel_sensor_lever_arm:
  vsm_x: 0.0
  vsm_y: 0.0
  vsm_z: 0.0

ins_initial_heading: auto  #auto #stored

ins_std_dev_mask:
  att_std_dev: 5
  pos_std_dev: 100

ins_output_type:
  output_location: MainAnt #MainAnt #POI1
  pos_std_dev: true
  att: true
  att_std_dev: true
  vel: true
  vel_std_dev: true

leap_seconds: 18

polling_period:
  pvt: 500
  rest: 500

reconnect_delay_s: 2

use_gnss_time: false

ntrip_settings:
  mode: off
  caster: 0
  caster_port: 0
  username: 0
  password: 0
  mountpoint: 0
  ntrip_version: v2
  send_gga: auto
  rx_has_internet: false
  rtcm_version: RTCMv2
  rx_input_corrections_tcp: 6666
  rx_input_corrections_serial: USB2

publish:
  gpgga: false
  gprmc: false
  gpgsa: false
  gpgsv: false
  pvtcartesian: false
  pvtgeodetic: true
  poscovcartesian: false
  poscovgeodetic: true
  atteuler: true
  attcoveuler: true
  # Available for both GNSS and INS receivers
  gpst : false
  navsatfix: true
  gpsfix: true
  pose: true
  diagnostics: false
  #INS messages
  insnavcart: true
  insnavgeod: true
  extsensormeas: true
  imusetup: true
  velsensorsetup: true
  exteventinsnavcart: false
  exteventinsnavgeod: false
  

